src/annotator/config/settings.js

```diff
@@ -129,11 +130,7 @@ export default function settingsFrom(window_) {
       return jsonConfigs[name];
     }
 
-    if (hasDefaultValue) {
-      return options.defaultValue;
-    }
-
-    return null;
+    return undefined;
```

Comment:
`undefined` is the implicit return value, but returning it explicitly here is good for clarity I think.

Reply:
my thoughts too